R/occumb.R

```diff
@@ -768,7 +768,27 @@ set_design_matrix <- function(formula, list_cov, omit_intercept = FALSE) {
         out <- tryCatch({
           stats::model.matrix(formula, data = list_cov)
         }, warning = function(w) {
-          stop(conditionMessage(w), "\n")
+          formula_terms <- formula |>
+            terms() |>
```

Comment:
Use `stats::terms()` instead of `terms()`; this will suppress a note in the `devtools::check()` results.